frontend/src/pages/tokens/index.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+	<div>
+		<Title>CCDScan | Tokens</Title>
+
+		<Table>
+			<TableHead>
+				<TableRow>
+					<TableTh>Token Id</TableTh>
+					<TableTh v-if="breakpoint >= Breakpoint.LG">Contract</TableTh>
+					<TableTh v-if="breakpoint >= Breakpoint.SM">{{
+						breakpoint >= Breakpoint.LG ? 'Metadata Url' : 'Metadata'
+					}}</TableTh>
+					<TableTh align="right"> Supply </TableTh>
+				</TableRow>
+			</TableHead>
+			<TableBody>
+				<TableRow
+					v-for="token in tokens"
+					:key="token.contractIndex + token.tokenId"
+				>
+					<TableTd>
+						<TokenLink :data="token.tokenId" :url="token.metadataUrl" />
+					</TableTd>
+					<TableTd>
+						{{ token.contractIndex }} / {{ token.contractSubIndex }}
+					</TableTd>
+
+					<TableTd v-if="breakpoint >= Breakpoint.LG">
+						<TokenLink
+							:data="token.metadataUrl"
+							:url="token.metadataUrl"
+							:length="25"
+							:suffix="'...'"
+						/>
+					</TableTd>
+
+					<TableTd v-if="breakpoint >= Breakpoint.SM" align="right">
+						<span class="numerical">
+							<TokenAmount
+								:symbol="token.metadata?.symbol"
+								:amount="token.totalSupply"
```

Comment:
Maybe I missed it , but you don't seem to be using the "unique" propery to show if this is fungible or not anywhere.

Reply:
its not being displayed in the token details sidebar